titus-server-master/src/main/java/com/netflix/titus/master/kubernetes/pod/KubePodConfiguration.java

```diff
@@ -135,6 +135,19 @@
     @DefaultValue("NONE")
     String getDisabledJobSpreadingPattern();
 
+    /**
+     * Job spreading skew adjustment factor, with the skew computed as job_size/alpha. Setting value <= 1 effectively
```

Comment:
minor: `s/with the skew/with the max skew/`

Reply:
also perhaps a good idea to mention this is job spreading on Nodes (not AZs, or some other topology dimension)